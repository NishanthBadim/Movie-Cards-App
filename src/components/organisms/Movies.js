import React, { Component } from 'react';
import MovieList from './MovieList';
import MovieService from '../../services/MovieService';

export default class Movies extends Component {

    constructor() {
        super();
        this.state = {
            movies: []
        };
    }

    componentDidMount() {
        this.setState(() => ({ movies: MovieService.getMovies() }));
    }

    render() {
        return (
            <div className="container-fluid" style={{ marginLeft: '-15px' }}>
                <MovieList movies={this.state.movies} />
            </div>
        );
    }
}