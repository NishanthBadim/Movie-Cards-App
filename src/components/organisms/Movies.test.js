import React from 'react';
import {create} from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect";
import Movies from './Movies'
import { __esModule } from 'react-modal-video';

jest.mock('./../organisms/MovieList', () => ({
    __esModule: true,
    default: () => { return <div></div>}
}))
test("Match the snapshot - Movies",() => {
    const input=create(<Movies />);
    expect(input.toJSON()).toMatchSnapshot();
})