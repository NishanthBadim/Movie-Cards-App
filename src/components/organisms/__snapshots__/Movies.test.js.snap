// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - Movies 1`] = `
<div
  className="container-fluid"
  style={
    Object {
      "marginLeft": "-15px",
    }
  }
>
  <div />
</div>
`;
